readme.txt

```diff
@@ -266,8 +266,8 @@ RVP
        error ::= "error" patno erno error.
      z ::= "\0" .
 
-     * RVP assumes that the last colon in a name separates the local part
-       from the namespace URI (it is what one gets if specifies `:' as
+     * RVP assumes that a pipa in a name separates the local part
```

Comment:
Typo: s/pipa/pipe/;
(I would just correct this myself but at the moment I cannot convince `git` to use my new PAT from GitHub.)

Reply:
Fixed, thanks :+1: